setup.py

```diff
@@ -122,7 +122,6 @@ def get_requirements(env):
             "auth_onelogin = sentry.auth.providers.saml2.onelogin",
             "auth_rippling = sentry.auth.providers.saml2.rippling",
             "auth_saml2 = sentry.auth.providers.saml2.generic",
-            "jira_ac = sentry_plugins.jira_ac",
```

Comment:
@ceorourke @wedamija yea, didn't run :( 

<img width="926" alt="Screen Shot 2021-09-08 at 11 57 09 AM" src="https://user-images.githubusercontent.com/8533851/132479238-72b02f34-88d0-416b-9112-a01d48966c4e.png">

Reply:
@wedamija if I delete the entire jira_ac folder in this PR then I'm assuming I don't need the migration ... since it'll be in a now non-existent folder, right? Is that safe? So the steps will be:

This PR:
* put jira_ac back into setup.py
* delete entire folder
Next PR: 
* migration to drop `JiraTenant` in regular Sentry migration folder
Final PR:
* remove jira_ac from setup.py

Reply:
If you delete the entire folder then you can also remove `jira_ac` from `setup.py`, in this step, since it's completely unused (I think you will hit errors loading the plugins otherwise)